app/javascript/controllers/typeahead_controller.js

```diff
@@ -0,0 +1,44 @@
+import { Controller } from "@hotwired/stimulus"
+import typeahead from 'typeahead-standalone';
+
+export default class extends Controller {
+  // Input where the typeahead will be hooked
+  static targets = ["input"]
+  // Details:
+  // src: the data source. Must return an array. URL must include a wildcard, which will be replaced
+  // with the value from the input
+  // wildcard: the string used as a wildcard
+  // identifier: the key from the returned objects used to display as suggestion in the typeahead dropdown
+  static values = {
+    src: { type: String, default: "/occupation_standards.json?q=QUERY"},
```

Comment:
Can we not put a default, and instead use the route name on the Rails view side? I like to avoid hardcoding any routes in JS since it is easier to end up with broken links if we change routing.

Reply:
@littleforest I'd like to put defaults values as examples in case someone wants to reuse the controller. We can keep the defaults with "dummy" values and add the real ones in the view. What are your thoughts on this?

Reply:
Sure that sounds good!